public/index.html

```diff
@@ -87,7 +87,19 @@
       href="manifest.json"
       style="--pwacompat-splash-font: 24px Virgil"
     />
-
+    <style>
+      body {
+        margin: 0;
+        --ui-font: system-ui, BlinkMacSystemFont, -apple-system, Segoe UI,
+          Roboto, Helvetica, Arial, sans-serif;
+        font-family: var(--ui-font);
+        color: var(--text-color-primary);
+        -webkit-text-size-adjust: 100%;
+        user-select: none;
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
```

Comment:
Shouldn't this be in the embed compo as well?

(either way, let's move it the existing `<style>` block below)

Reply:
no this is for excalidraw web app. The host app doesn't need it we shouldn't style body of host as well so everything for the host app will be styled under namespace excalidraw

Reply:
true